tests/nightly-kickoff.sh

```diff
@@ -0,0 +1,332 @@
+
+cd "$(git rev-parse --show-toplevel)"
+
+tests=${*#${PWD}/}
+
+input=$(wget -O - https://vmware.bintray.com/vic-repo |tail -n5 |head -n1 |cut -d':' -f 2 |cut -d'.' -f 3| cut -d'>' -f 2)
+echo "https://console.cloud.google.com/m/cloudstorage/b/vic-ci-logs/o/functional_logs_"$input".zip?authuser=1"
+echo "Downloading bintray file"
+wget https://vmware.bintray.com/vic-repo/$input.tar.gz
+echo "Removing VIC directory if present"
+rm -rf vic
+echo "Extracting .tar.gz"
+tar xf $input.tar.gz
+echo "Deleting .tar.gz vic file"
+rm $input.tar.gz
+
+drone exec --trusted --yaml <(cat <<CONFIG
+---
+clone:
+  path: github.com/vmware/vic
+  tags: true
+
+build:
+  integration-test:
+    image: vmware-docker-ci-repo.bintray.io/integration/vic-test:1.8
+    pull: true
+    environment:
+      BIN: vic
+      GOPATH: /drone
+      SHELL: /bin/bash
+      DOCKER_API_VERSION: "1.21"  
+      LOG_TEMP_DIR: install-logs     
+      TEST_TIMEOUT: $TEST_TIMEOUT
+      GS_PROJECT_ID: $GS_PROJECT_ID
+      GS_CLIENT_EMAIL: $GS_CLIENT_EMAIL
+      GS_PRIVATE_KEY: $GS_PRIVATE_KEY
+      NIMBUS_USER: $NIMBUS_USER
+      NIMBUS_PASSWORD: $NIMBUS_PASSWORD
+      NIMBUS_GW: $NIMBUS_GW
+      SLACK_URL: $SLACK_URL
+      GS_PROJECT_ID: $GS_PROJECT_ID
+      GS_CLIENT_EMAIL: $GS_CLIENT_EMAIL
+      GS_PRIVATE_KEY: $GS_PRIVATE_KEY
+    commands:
+       - pybot tests/manual-test-cases/Group5-Functional-Tests/5-1-Distributed-Switch.robot
```

Comment:
Sure, I know we discussed this approach. I wanted to get the initial draft for feedback. I will make it dynamic and also have individual yaml files.